### DISCUSSION TITLE
After update to macOS Monterey 12.5.1 Volume Control is not working.

### DISCUSSION THREAD

**Discussion by ceadmond** [MonitorControl/MonitorControl#1165] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

After update to macOS Monterey 12.5.1 volume control is not working on my PHILIPS 279P1.

### Expected behavior

Volume control is not working with no sound.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.5.1 Monterey
- Mac model: MacBook Pro (14-inch, 2021)
- MonitorControl version: v4.1.0
- Monitor(s): PHL 279P1
- Monitor Cable(s)/Connection(s): MacBook Pro -> Baseus USB 4 Dock Cable -> PHL 279P1
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Do you have this issue happen when the display is directly connected without a dock? Does brightness control work between 50%-100% brightness (via DDC)?

**Comment by niklasnolte** [comment]
i have the problem when directly connected via usbc. I can slide the volume bar, but nothing changes.
Mine is a LENOVO P27h-10

**Comment by rkieru** [comment]
I'm facing a similar issue, specs below:

- macOS v12.5.1 (21G83)
- MacBook Pro (14-inch, 2021), Apple M1 Pro
- Dell S2721QS
- Display Port to USB-C connection

I can control brightness with my keyboard controls just fine, but volume is not honored.

**Comment by waydabber** [comment]
If brightness control / DDC fully works but volume control does not, then the display simply does not support changing the volume via DDC. A high number of displays have a firmware like that (even mine works that way, even though it's a rather expensive Lenovo CreatorXtreme). Not much to do about that.

**Comment by rattrayalex** [comment]
Darn, does anybody know of a way to play sound through a PHL 272P or 279P Philips Brilliance USB-C monitor from an M1 or M2 Macbook Air or Macbook Pro running MacOS Monterey 12.6? At all? I came from [this thread](https://discussions.apple.com/thread/252778777) which implied that it might help, but still, no sound at all…

**Reply by rattrayalex** [reply]
Note that this app _does_ enable me to change the volume percentage that PHL displays, so it _is_ changing the volume – just that no sound is coming out. @waydabber does that help with debugging at all?

**Comment by user858753257** [comment]
> If brightness control / DDC fully works but volume control does not, then the display simply does not support changing the volume via DDC. A high number of displays have a firmware like that (even mine works that way, even though it's a rather expensive Lenovo CreatorXtreme). Not much to do about that.

Have you infos about the Dell S-Series  ? 

I have an S3423DWC . I can preee the keys and the popup is coming .

But the speaker isn't changed
